packages/@react-spectrum/alert/package.json

```diff
@@ -5,6 +5,10 @@
   "license": "Apache-2.0",
   "private": true,
   "main": "dist/Alert.js",
+  "exports": {
+    "import": "./dist/Alert.module.js",
```

Comment:
Is this package being built wrong?

Reply:
i believe this is correct, it's using the same strings as module and main, I don't know why this file is different from others by including the name in the filename

Reply:
I was just pointing out that a couple filenames are different.